Dockerfile

```diff
@@ -11,7 +11,7 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-FROM golang:1.17.5-alpine3.14
+FROM golang:1.20.8-alpine3.18
```

Comment:
Any reason to choose 1.20 over 1.21?

(1.20 is still supported so it's definitely not an issue)

Reply:
Good question - I chose 1.20 just because I am still on 1.20 locally 😂